chart/fcrepo/templates/deployment.yaml

```diff
@@ -0,0 +1,109 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: {{ include "fcrepo.fullname" . }}
+  labels:
+    {{- include "fcrepo.labels" . | nindent 4 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      {{- include "fcrepo.selectorLabels" . | nindent 6 }}
+  strategy:
+    type: Recreate
+  template:
+    metadata:
+    {{- with .Values.podAnnotations }}
+      annotations:
+        {{- toYaml . | nindent 8 }}
+    {{- end }}
+      labels:
+        {{- include "fcrepo.selectorLabels" . | nindent 8 }}
+    spec:
+      {{- with .Values.imagePullSecrets }}
+      imagePullSecrets:
+        {{- toYaml . | nindent 8 }}
+      {{- end }}
+      restartPolicy: Always
+      serviceAccountName: {{ include "fcrepo.serviceAccountName" . }}
+      securityContext:
+        {{- toYaml .Values.podSecurityContext | nindent 8 }}
+      volumes:
+        - name: data
+          persistentVolumeClaim:
+            claimName: {{ template "fcrepo.fullname" . }}
+      initContainers:
+        - name: "remove-lost-found"
+          image: "busybox:1.32.0"
+          command:
+            - rm
+            - -fr
+            - /data/lost+found
+          volumeMounts:
+            - name: data
+              mountPath: /data
+        {{- if not .Values.postgresql.enabled }}
+        {{- if .Values.fcrepoSecretName }}
+        - name: "init-db"
+          image: {{ .Values.postgresql.image.repository | default "bitnami/postgresql" }}:{{ .Values.postgresql.image.tag | default "latest" }}
+          command:
+            - sh
+            - -c
+            - export PGPASSWORD=$DATABASE_PASSWORD PGHOST=$DATABASE_HOST PGUSER=$DATABASE_USER
```

Comment:
can these be set in an `environment` block below?

Reply:
probably. i'm going to suggest we merge and tweak from there.

this is now deploying successfully and i'm keen on being able to work both on improving the chart and on getting some fashion of CD in place. `nurax`'s current issues are worrisome since only a few people can work on its infra.